### DISCUSSION TITLE
SelfHosted, iOS, Error - We were unable to process your request. Please try again or contact us.

### DISCUSSION THREAD

**Discussion by santiagochen** [dani-garcia/vaultwarden#4573] [Q&A]
Hi, just putting this out here in case somebody has encountered or solved this issue.

Just recently (in the last week or so), I am no longer able to use Bitwarden on my android phone. Everytime I try to autofill the fields, it shows this error "We were unable to process your request. Please try again or contact us."

Iphone Client fails, but it is ok on the browser; I found I am alone; there are a lot of people reporting the issues;

**Comment by MrDuartePT** [comment]
I have my instance on a domain protect by cloudflare, and I can’t sync or login using the iOS Bitwarden app if the Cloudflare proxy is on in the Bitwarden hostname. 

I notice when I enable Cloudflare proxy my hostname uses cloudflare universal cert, instead of my let’s encrypt one, but it strange that this errors only happen on iOS.

**Reply by MrDuartePT** [reply]
@santiagochen pls try the Bitwarden beta app it semms to work for me with cloudflare proxy enable: https://testflight.apple.com/join/S4OQEYYQ

I test both with latest (1.31.0) and testing vaultwarden servers
